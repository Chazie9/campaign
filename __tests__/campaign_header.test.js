import React from 'react';
import { shallow } from 'enzyme';
import CampaignHeader from '../client/src/components/campaign_header.jsx';

describe('Campaign Header', () => {
    it('should be defined', () => {
      expect(CampaignHeader).toBeDefined();
    });
   
    it('should render correctly', () => {
      const tree = shallow(
        <CampaignHeader name='yoyo test' />
      );
      expect(tree).toMatchSnapshot();
    });
});