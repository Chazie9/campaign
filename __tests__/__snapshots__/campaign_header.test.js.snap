// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Campaign Header should render correctly 1`] = `
<div
  className="user-and-title-info"
>
  <div
    className="creatorInfo"
  >
    <div
      className="creator"
    >
      <img
        alt="user image"
        src="test-file-stub"
      />
    </div>
    <div
      className="creator"
    >
      by: Chas Harring
    </div>
    <div
      className="creator"
    >
      5 projects
    </div>
    <div>
      <button>
        Follow
      </button>
    </div>
  </div>
  <div
    className="title-and-tagline-container"
  >
    <div
      className="title-and-tagline"
    >
      <h1>
        This is a big ol bag
      </h1>
      <h4>
        this is the best biggest ol bag
      </h4>
    </div>
  </div>
</div>
`;
